doc/source/user_guide/model-mapping.rst

```diff
@@ -0,0 +1,149 @@
+  
+  * *model* and 
+  * at least one of *native_regions* and *common_regions*
+
+* Optional properties are:
+  * *exclude_regions*
+
+*  *model* (str or list of str): the model name(s) for which the mapping applies.
+*  *native_regions* (list): a list of model native regions serves as
+   a selection as to which regions to keep.
+
+   *  In the above example *region_a* is to be renamed to
+      *alternative_name_a*. This is done by defining a key-value pair
+      of *model_native_name: new_name*.
+   *  *region_b* is selected but the name is not changed.
+   *  Assuming *model_a* also defines a third region *region_c*,
+      since it is not mentioned it will be **dropped** from the data.
+
+*  *common_regions* (list): list of common regions which will be computed as aggregates.
+   They are defined as list entries which themselves have a list of constituent regions.
+   These constituent regions must be model native regions.
+
+   The names of the constituent regions **must** refer to the **original** model native
+   region names. In the above example *region_a* and *region_b* and **not**
+   *alternative_name_a*.
+
+* *exclude_regions* optional (list of str): If input data for region processing contains
+  regions which are not mentioned in *native_regions*, in *common_regions* (as the name
+  of a common region or a constituent region) an error will be raised. This is a
+  safeguard against silently dropping regions which are not in named in *native_regions*
+  or *common_regions*. 
+  
+  If regions are to be excluded, they can be explicitly named in the *exclude_regions*
+  section which causes their presence to no longer raise an error.
+
+
+Region aggregation
+------------------
+
+In order to illustrate how region aggregation is performed, consider the following model
+mapping:
+
+.. code:: yaml
+
+   model: model_a  
+   common_regions:
+     - common_region_1:
+       - region_a
+       - region_b
+
+If the data provided for region aggregation contains results for *common_region_1* they
+are compared and combined according to the following logic:
+
+1. If a variable is **not** reported for *common_region_1*, it is calculated through
+   region aggregation of regions *region_a* and *region_b*.
+2. If a variable is **only** reported for *common_region_1* level it is used directly.
+3. If a variable is is reported for *common_region_1* **as well as** *region_a* and
+   *region_b*. The **provided results** take **precedence** over the aggregated ones.
+   Additionally, the aggregation is computed and compared to the provided results. If
+   there are discrepancies, a warning is written to the logs.
+   
+   .. note::
+
+      Please note that in case of differences no error is raised. Therefore it is
+      necessary to check the logs to find out if there were any differences. This is
+      intentional since some differences might be expected.
+
+The process of combining aggregated and model native results is dubbed
```

Comment:
Is this name really used anywhere, other than in some tests? If not, I suggest removing this entirely.

Reply:
Good point, it's not, let's leave it out then.